lemminx-maven/pom.xml

```diff
@@ -102,6 +102,22 @@
 
 	<build>
 		<plugins>
+	<!-- This is a workaround for CI because it looks like CI cannot connect 

```

Comment:
Seems like CI is still failing while including these plugins in the build. Maybe they should be included in the project's dependencies instead?